cypress/support/application-stub.tsx

```diff
@@ -0,0 +1,17 @@
+import '../../src/app/[locale]/globals.css';
+import {QueryClient, QueryClientProvider} from '@tanstack/react-query';
+import {ReactNode} from 'react';
+
+export default function ApplicationStub({children}: {children: ReactNode}) {
```

Comment:
Is it possible to not use a `default export` but only an `export`? This makes it consistent with other parts of the codebase, and allows for future expansion (e.g. if we want to add another function to this file).

Reply:
I tend to use only `export` if I think multiple things will be exported. For a file with one component, I find `default export` more clear, to send the message it's just one component.

Do you want only `export` for all files? Because then I need to change this for more than just this file.

Reply:
You're right! This is not some regular code - this is a React component. In this case I would indeed use a `default export` (because that's the React convention).